### ISSUE TITLE
where are (or how to build) the generated libraries?

### ISSUE THREAD

**Issue by qtxo** [googleapis/google-api-java-client#1264]
The official web page https://developers.google.com/api-client-library/java/ still references the old version 1.25.0.

The latest official generated library for Drive API
```
  <dependency>
      <groupId>com.google.apis</groupId>
      <artifactId>google-api-services-drive</artifactId>
      <version>v2-rev361-1.25.0</version>
    </dependency>
```

But, I am using version 1.28.0 I found in maven repo for some libs:
```
  <dependency>
      <groupId>com.google.apis</groupId>
      <artifactId>google-api-services-drive</artifactId>
      <version>v2-rev20190501-1.28.0</version>
    </dependency>
```
What is the correct and official version of the generated libraries? 

Or should we generate our own?

**Comment by chingor13** [comment]
Apologies, we are in the process of updating those docs to point to the real latest versions.

The source code for the newest versions exists https://github.com/googleapis/google-api-java-client-services

**Comment by qtxo** [comment]
> The source code for the newest versions exists https://github.com/googleapis/google-api-java-client-services

I can't find the sources for Admin Directory API, they exist in a private repo?

**Comment by chingor13** [comment]
The missing Admin APIs from the list was fixed in https://github.com/googleapis/google-api-java-client-services/pull/2607. Links from the README to the list of services is being fixed in #1353
